spec/rails_helper.rb

```diff
@@ -75,4 +75,5 @@
   end
 end
 
-OmniAuth.config.test_mode = true
\ No newline at end of file
+OmniAuth.config.test_mode = true
+OmniAuth.config.silence_get_warning = true
```

Comment:
what does this do?

Reply:
its a way of mocking out OmniAuth so we can test user creation without making a network call
the second line silence the error for allowing get requests with OAuth. needed to work with the github OAuth provider